aten/src/ATen/native/cuda/SortingRadixSelect.cuh

```diff
@@ -318,7 +318,6 @@ __device__ void radixSelect(
 
   // We start at the most significant digit in our radix, scanning
   // through to the least significant digit
-#pragma unroll
```

Comment:
I found that the `#pragma unroll` here contributes most to the binary size of topk kernel, which could be reduced from 68M to 11M after removing this line.

```
# master 
-rw-rw-r--  1 richard richard  61M Jan 17 13:49 TensorTopK.sm_86.cubin

# master after remove pragma unroll
-rw-rw-r--  1 richard richard  11M Jan 17 14:01 TensorTopK.sm_86.cubin
```

The benchmark here shows that the performance is almost identical after removing this `#pragma unroll`
https://docs.google.com/spreadsheets/d/1nYHiCBJ-dbliO0lzS8uQSB3wbPn8hBlWALSjS7bS09Q/edit?usp=sharing

Benchmark script:
```py
import torch
import time
import torch
import pandas as pd


dtype = torch.half

k = 2000
dim = 1
largest = True
sorted = False
torch.manual_seed(1)


def benchmark(D1, D2, k, dtype):
    x = torch.randn((D1, D2), dtype=dtype, device="cuda", requires_grad=True)

    torch.cuda.synchronize()
    start_time = time.time()
    for _ in range(1000):
        values_ref, indices_ref = x.topk(k=k, dim=dim, sorted=sorted, largest=largest)
    torch.cuda.synchronize()
    t = (time.time() - start_time) / 1000
    print(f'{t:.10f}')

    return t


data = []

for d1 in list(range(10, 140, 30))+[600, 660]:
    D2 = [2**11, 2**12, 2**13, 2**14, 2**15, 2**16, 2**17] if d1<600 else [2**11, 2**12]
    for d2 in D2:
        print(f"----------------- D1 = {d1}, D2 = {d2} -----------------")
        t = benchmark(d1, d2, k, dtype)
        data.append([d1, d2, k, t])

df = pd.DataFrame(data=data, columns=['D1', 'D2', 'k', 'time'])
print(df)
df.to_csv('benchmark.csv')
```

Reply:
This is amazing! Can you also check that it doesn't regress `sort` performance? This function is used for small non-stable sorts (slices less than 2048, stable sort=false or something like this)

Reply:
Oh, sorry, I was wrong, it's not used by sort, it's only used by topK/kthvalue/median, so yeah, this benchmark is good.

Reply:
ok! kthvalue is also benchmarked here: https://docs.google.com/spreadsheets/d/1nYHiCBJ-dbliO0lzS8uQSB3wbPn8hBlWALSjS7bS09Q/edit#gid=755018695, there is no performance changes too.

Reply:
I've done a bit more benchmarking, and on V100 sometimes I see up to 20% difference (that's on current master, w/o this PR)
https://gist.github.com/ngimel/16c0f00963ba099eb9b8025cc83dd778
but I guess it's still acceptable given the reduction in context size and build time

Reply:
Just added another columns after `change largest as func args`, https://docs.google.com/spreadsheets/d/1nYHiCBJ-dbliO0lzS8uQSB3wbPn8hBlWALSjS7bS09Q/edit#gid=0, there is a bit performance regress for `float` (average 0.927, worst 0.81), but I hope it is acceptable.